src/main/java/step3/study/domain/Car.java

```diff
@@ -0,0 +1,34 @@
+package step3.study.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Car {
+    private String distance = "";
+    private List<String> recodeDistance;
+    private static int MOVE_CONDITION = 4;

```

Comment:
```suggestion
    private static final int MOVE_CONDITION = 4;
```

상수의 경우에는 꼭 final 을 붙여줄 수 있도록 해주세요